cluster-management/setup/mounting-storage/index.md

```diff
@@ -64,6 +64,46 @@ coreos:
 
 Notice that we're starting both units at the same time and using the power of systemd to work out the dependencies for us. In this case, `var-lib-docker.mount` requires `format-ephemeral.service`, ensuring that our storage will always be formatted before it is mounted. Docker will refuse to start otherwise.
 
+## Creating and Mounting a btrfs Volume File
+
+CoreOS versions 6xx.0.0 and later use ext4 + overlayfs to provide a layered filesystem for the root partition.
```

Comment:
Please change `6xx.0.0` to:

    [561.0.0](https://coreos.com/releases/#561.0.0)